hypertrace-trace-enricher/trace-reader/src/main/java/org/hypertrace/trace/accessor/entities/TraceEntityAccessorBuilder.java

```diff
@@ -11,6 +13,7 @@ public class TraceEntityAccessorBuilder {
   private final EntityDataClient entityDataClient;
   private final CachingAttributeClient attributeClient;
   private Duration entityWriteThrottleDuration = Duration.ofSeconds(15);
+  private Set<String> excludedEntityTypes = new HashSet<>();
```

Comment:
emptyset or similar. builder data shouldn't be mutable except through builder interfaces